src/Libraries/CoreNodesUI/Input/DoubleSlider.cs

```diff
@@ -1,113 +1,95 @@
         {
-            get { return _max; }
-            set
-            {
-                _max = value;
-                if (_max < Value)
-                    Value = _max;
-                RaisePropertyChanged("Max");
-            }
+            var rhs = AstFactory.BuildDoubleNode(Value);
+            var assignment = AstFactory.BuildAssignment(GetAstIdentifierForOutputIndex(0), rhs);
+
+            return new[] { assignment };
         }
 
-        private double _min;
-        public double Min
+        protected override double DeserializeValue(string val)
         {
-            get { return _min; }
-            set
+            try
             {
-                _min = value;
-                if (_min > Value)
-                    Value = _min;
-                RaisePropertyChanged("Min");
+                return Convert.ToDouble(val, CultureInfo.InvariantCulture);
             }
-        }
-
-        #region Serialization/Deserialization Methods
-
-        protected override void SerializeCore(XmlElement element, SaveContext context)
-        {
-            base.SerializeCore(element, context); // Base implementation must be called.
-
-            XmlElement outEl = element.OwnerDocument.CreateElement("Range");
-            outEl.SetAttribute("min", Min.ToString(CultureInfo.InvariantCulture));
-            outEl.SetAttribute("max", Max.ToString(CultureInfo.InvariantCulture));
-            element.AppendChild(outEl);
-        }
-
-        protected override void DeserializeCore(XmlElement nodeElement, SaveContext context)
-        {
-            base.DeserializeCore(nodeElement, context); //Base implementation must be called.
-
-            foreach (XmlNode subNode in nodeElement.ChildNodes)
+            catch
             {
-                if (!subNode.Name.Equals("Range"))
-                    continue;
-
-                double min = Min;
-                double max = Max;
-
-                if (subNode.Attributes != null)
-                {
-                    foreach (XmlAttribute attr in subNode.Attributes)
-                    {
-                        if (attr.Name.Equals("min"))
-                            min = Convert.ToDouble(attr.Value, CultureInfo.InvariantCulture);
-                        else if (attr.Name.Equals("max"))
-                            max = Convert.ToDouble(attr.Value, CultureInfo.InvariantCulture);
-                        else if (attr.Name.Equals("value"))
-                            Value = Convert.ToDouble(subNode.InnerText, CultureInfo.InvariantCulture);
-                    }
-                }
-
-                Min = min;
-                Max = max;
+                return 0;
             }
         }
 
-        #endregion
+        protected override string SerializeValue()
+        {
+            return Value.ToString(CultureInfo.InvariantCulture);
+        }
 
         protected override bool UpdateValueCore(string name, string value, UndoRedoRecorder recorder)
```

Comment:
Unfortunately not. Again, because the base class has a generic type, we cannot call `Value.ToString(CultureInfo.InvariantCulture)` because that ToString override is only available on numeric types. It's not until that actual Double or Integer slider implementation where we specify the type on the base class that we can do this.